src/main/java/net/minecraftforge/fml/common/network/FMLOutboundHandler.java

```diff
@@ -43,9 +44,10 @@
 import javax.annotation.Nullable;
 
 public class FMLOutboundHandler extends ChannelOutboundHandlerAdapter {
-    public static final AttributeKey<OutboundTarget> FML_MESSAGETARGET = AttributeKey.valueOf("fml:outboundTarget");
+    public static final AttributeKey<IOutboundTarget> FML_MESSAGETARGET = AttributeKey.valueOf("fml:outboundTarget");
```

Comment:
I believe it's not safe to change this type, any mod referencing it will break.
If you want to add this, it must be done separately for the 1.12 update as a breaking change.